pyunifiprotect/unifi_data.py

```diff
@@ -250,6 +318,30 @@ def event_from_ws_frames(state_machine, minimum_score, action_json, data_json):
 
     return camera_id, processed_event
 
+#TODO: implement light_update_from_ws_frames
+
+def light_update_from_ws_frames(state_machine, host, action_json, data_json):
+    """Convert a websocket frame to internal format."""
+
+    if action_json["modelKey"] != "light":
+        raise ValueError("Model key must be light")
+
+    light_id = action_json["id"]
+
+    if not state_machine.has_camera(light_id):
+        _LOGGER.debug("Skipping non-adopted light: %s", data_json)
+        return None, None
+
+    light = state_machine.update(light_id, data_json)
+
+    if data_json.keys().isdisjoint(CAMERA_KEYS):
```

Comment:
```suggestion
    if data_json.keys().isdisjoint(LIGHT_KEYS):
```

and add `LIGHT_KEYS` to list the keys we care about.